report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,38 +1,60 @@
 \section{Problem Definition}\label{sec:problem_definition}
 The primary objective of this research is to enhance computational methods for the accurate and robust quantification of chemical compositions using \gls{libs} data.
-As previously introduced, quantifying chemical compositions from \gls{libs} spectral data poses significant challenges, including high dimensionality and multicollinearity of the data, as well as matrix effects.
-Here, we further examine these challenges:
-\begin{itemize}
-    \item \textbf{Data Dimensionality and Collinearity:} The high dimensionality of spectral data, coupled with multicollinearity—where multiple spectral features may exhibit strong correlations—complicates the modeling and analysis\cite{andersonImprovedAccuracyQuantitative2017}.
-    \item \textbf{Matrix Effects:} As noted in the introduction, matrix effects refer to any effect that can cause variations in the intensity of emission lines of an element independent of the concentration of that element. The presence of different background elements can alter the emission intensities and pose significant challenges in accurately quantifying elemental concentrations. The spectra are complex due to the interaction of multiple physical processes including the coupling process between the laser photons and the target, self-absorption of optical emission lines within the plasma, recombination of elements into molecules, and collisional interactions in the plasma\cite{cleggRecalibrationMarsScience2017, andersonImprovedAccuracyQuantitative2017}.
-    \item \textbf{Data Availability:} As highlighted earlier, due to the high cost of data collection, datasets are often small, which can limit the generalizability and robustness of the models\cite{p9_paper}.
-\end{itemize}
-
-The complexities presented by these challenges require the creation of advanced computational models designed to address and alleviate such issues, thereby enhancing the accuracy and reliability of chemical composition analysis with \gls{libs} data.
 
-The input to such models consists of \gls{libs} spectral data, which includes intensity readings across a spectrum of wavelengths.
-This data is in the form of Clean, Calibrated Spectra\cite{andersonImprovedAccuracyQuantitative2017}, the output of level 1 processing as described by \citet{wiensPreflightCalibrationInitial2013}.
-The wavelength intensities are quantified in units of \\ photon/shot/mm\textsuperscript{2}/sr/nm.
+The input to such models consists of \gls{libs} spectral data, which includes intensity readings across a spectrum of wavelengths. This data is in the form of Clean, Calibrated Spectra\cite{andersonImprovedAccuracyQuantitative2017}, the output of level 1 processing as described by \citet{wiensPreflightCalibrationInitial2013}. The wavelength intensities are quantified in units of \\ photon/shot/mm\textsuperscript{2}/sr/nm.
 
 Formally, we have:
+\newcounter{listitem}
```

Comment:
I've made a suggestion to remove spectrometer matrix, use $n_o$ to denote oxide count, use $\lambda$ to denote wavelengths. Please see the commit with SHA `762791bfa9f4`. Feel free to reverse it if you disagree.

While I think the use of cardinality notation is clever, I believe it isn't standard notation, and therefore isn't preferable.
Please let me know your thoughts.

Cardinality notation (|o|) typically denotes the number of elements in a set. The index $o$ represents different oxides, but  $o$ itself is not a set; it's an index variable ranging over a set of oxides. So it might confuse readers who expect it to signify the size of a set, not the range of an indexing variable.

Reply:
Agree